core/io/marshalls.h

```diff
@@ -215,6 +217,8 @@ class EncodedObjectAsID : public RefCounted {
 Error decode_variant(Variant &r_variant, const uint8_t *p_buffer, int p_len, int *r_len = nullptr, bool p_allow_objects = false, int p_depth = 0);
 Error encode_variant(const Variant &p_variant, uint8_t *r_buffer, int &r_len, bool p_full_objects = false, int p_depth = 0);
 
+Error to_raw_bytes(const Variant &p_variant, PackedByteArray &out_sink);
```

Comment:
```suggestion
Error to_raw_bytes(const Variant &p_variant, PackedByteArray &r_out_sink);
```
We use this prefix for return values